### PR TITLE
[Merged by Bors] - chore(SimpleGraph/Regularity): Don't use `Classical`

### PR DESCRIPTION
**Pull request by YaelDillies**
The classical decidability instances were found in theorem statements, making them harder to use.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Combinatorics/SimpleGraph/Regularity/Increment.lean [hunk 1]
```diff
@@ -105,7 +108,7 @@ private theorem distinctPairs_increment :
 
 private lemma pairwiseDisjoint_distinctPairs :
     (P.parts.offDiag.attach : Set {x // x ∈ P.parts.offDiag}).PairwiseDisjoint
-      (distinctPairs G ε hP) := by
```
Comment (eric-wieser) [inline Mathlib/Combinatorics/SimpleGraph/Regularity/Increment.lean:108]: Is this argument ordering desirable? I think it's likely that the original non-use of `variable`s was deliberate to affect the order.

Comment (eric-wieser) [reply Mathlib/Combinatorics/SimpleGraph/Regularity/Increment.lean:108]: Oh nevermind, it's `private`

**Review by eric-wieser — APPROVED** [review]
bors merge

File: Mathlib/Combinatorics/SimpleGraph/Regularity/Increment.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Combinatorics/SimpleGraph/Regularity/Increment.lean:108]: I will also note that "original" here means "written by 2021 me", so I don't feel too bad for impeding on myself!

**Pull request closed without merging**